plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1706,8 +1709,9 @@ public void testStatsSplitPruningBasedOnSepCreatedCheckpointOnTopOfCheckpointWit
         assertUpdate("DROP TABLE " + tableName);
     }
 
-    @Test
-    public void testVacuum()
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
```

Comment:
We usually avoid `@ParameterizedTest`. https://trino.io/docs/current/develop/tests.html#avoid-data-providers-and-parametric-tests